src/main/java/com/moodybluez/enterprise/dao/DateDAO.java

```diff
@@ -0,0 +1,51 @@
+package com.moodybluez.enterprise.dao;
+
+import com.moodybluez.enterprise.dto.Date;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class DateDAO implements IDateDAO{
+
+    private Map<Integer, Date> dates = new HashMap<>();
+
+    @Override
+    public boolean createEntry(Date date) {
+        dates.put(date.getDateID(), date);
+        if (dates.containsKey(date.getDateID())) return true;
```

Comment:
You can shorten this with:

return dates.ContainsKey(date.getDateID())

That will give you the same result with less typing.